tweeter.py

```diff
@@ -99,7 +99,25 @@ def stats():
 
     return stat_string_1 + stat_string_2 + stat_string_3 + stat_string_4
 
-#"Tweeter Stats - Number of adjectives: <num_adj>, Number of objects: <num_obj>, Number of antecedents: <num_ant>, Number of postcedents: <num_post>"
+# 
+
+@app.route("/reversedtweet")
+def reversedtweet():
+    adj_index = random.randint(0, len(adj_list) - 1)
+    obj_index = random.randint(0, len(obj_list) - 1)
+    ant_index = random.randint(0, len(antecedent_list) - 1)
+    post_index = random.randint(0, len(postcedent_list) - 1)
+    tweet_subj = "{} {}".format(adj_list[adj_index], obj_list[obj_index])
+    tweet = "{} {}".format(antecedent_list[ant_index].format(tweet_subj), postcedent_list[post_index])
+
+    reversedtweet  = ""
```

Comment:
In most programming languages large variables that involve multiple words use camel case for readability, for example: `aLargeVariable`

The convention when writing Python however is to used snek case so `reversedtweet` should be named `reversed_tweet`.